packages/beacon-node/src/api/impl/validator/index.ts

```diff
@@ -335,6 +338,35 @@ export function getValidatorApi({
       );
   }
 
+  async function updateDefaultGraffiti(): Promise<void> {
+    defaultGraffiti = await getDefaultGraffiti();
+  }
+
+  async function getDefaultGraffiti(): Promise<string> {
+    const consensusCode = ClientCode.LS;
+    const consensusCommit = opts.version ?? "";
+
+    const lodestarClientVersion = {
+      code: consensusCode,
+      name: "Lodestar",
+      version: opts.version ?? "",
+      commit: consensusCommit,
+    };
+
+    const executionClientVersions = await chain.executionEngine
+      .getClientVersion(lodestarClientVersion)
+      .catch((_) => []);
+
+    if (executionClientVersions.length !== 0) {
+      const executionCode = executionClientVersions[0].code;
+      const executionCommit = executionClientVersions[0].commit;
+
+      return `${executionCode}|${executionCommit.slice(0, 2)}|${consensusCode}|${consensusCommit.slice(0, 2)}`;
+    }
+
+    return "";
```

Comment:
we should still include our info in the graffiti even if we don't get a good response from the execution client